zerocopy-derive/src/lib.rs

```diff
@@ -1076,7 +1076,17 @@ enum Trait {
 
 impl ToTokens for Trait {
     fn to_tokens(&self, tokens: &mut TokenStream) {
-        let ident = Ident::new(&format!("{:?}", self), Span::call_site());
+        let s = match self {
```

Comment:
Can you add a comment here explaining why this verbosity is necessary and just using Debug isn't sufficient?

Reply:
Done. PTAL